rust/arrow/Cargo.toml

```diff
@@ -47,7 +47,7 @@ regex = "1.3"
 lazy_static = "1.4"
 packed_simd = { version = "0.3.4", optional = true, package = "packed_simd_2" }
 chrono = "0.4"
-flatbuffers = "^0.8"
+flatbuffers = "=0.8.3" # 0.8.4 introduces const generics
```

Comment:
this may be accidentally included from  https://github.com/apache/arrow/pull/10081